javascript-data-model/order-history.js

```diff
@@ -0,0 +1,54 @@
+const orderHistory = [
+  {
+    type: 'Book',
+    name: '"Javascript for impatient programmers"',
+    author: 'Rauschmayer, Dr. Axel',
+    initialCost: '$31.55',
+    finalCost: '$34.00',
+    orderDate: 'August 4, 2020',
+    deliveryDate: 'August 8, 2020',
+    returnWindowClose: 'September 7, 2020',
+    shipTo: 'JS Masher',
+    orderNumber: '114-3941689-8772232'
+  },
+  {
+    type: 'Book',
+    name: '"The Timeless Way of Building"',
+    author: 'Alexander, Christopher',
+    initialCost: '$41.33',
+    finalCost: '$44.53',
+    orderDate: 'July 19, 2020',
+    deliveryDate: 'July 20, 2020',
+    returnWindowClose: 'August 19, 2020',
+    shipTo: 'JS Masher',
+    orderNumber: '113-9984268-1280257'
+  },
+  {
+    type: 'Electronic',
+    name: '"Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for WII U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter"',
+    author: 'N/A',
+    initialCost: '$15.98',
+    finalCost: '$17.22',
+    orderDate: 'July 4, 2020',
+    deliveryDate: 'July 7, 2020',
+    returnWindowClose: 'August 5, 2020',
+    shipTo: 'JS Masher',
+    orderNumber: '114-2875557-9059409'
+  },
+  {
+    type1: 'Electronic',
+    type2: 'Book',
+    name1: '"GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)"',
+    name2: '"The Art of Sql"',
+    author1: 'N/A',
+    author2: 'Faroult, Stephane',
+    initialCost1: '$94.95',
+    initialCost2: '$33.99',
+    finalCost: '$138.93',
+    orderDate: 'July 3, 2020',
+    deliveryDate: 'July 5, 2020',
+    returnWindowClose: 'August 4, 2020',
+    shipTo: 'JS Masher',
+    orderNumber: '113-2883177-2648248'
+  }
```

Comment:
I see what you're trying to do, this one is tricky -- the goal behind a data model is to have some consistency with every order (re: object)  in your `orderHistory` array.

I would recommend having an `items` property that contains an array with the same structure. Let's try and refactor it so that the orders here follow a repeatable model.
Even if your order only has one `item`, you can still have an `items` property that has an array with only one index.